exporter/solarwindsexporter/README.md

```diff
@@ -13,25 +13,24 @@ SolarWinds Exporter is a convenience wrapper around [OTLP gRPC Exporter](https:/
 
 ## Getting Started
 
-You just need to include the SolarWinds Exporter in your exporter definitions and provide the following minimal configuration:
+You just need to include the SolarWinds Exporter in your exporter definitions and no additional configuration is needed. It always needs to be used together with the [Solarwinds Extension](../../extension/solarwindsextension).
 
 ```yaml
 exporters:
+  solarwinds: {}
```

Comment:
nit: not sure how to interpret the `{}` from reader perspective

Reply:
to me yes, as I tend to more YAML-puritan notation, but if this is somehow standardized 'tis a gray zone and I'll defer to your choice 👍